vmdb/app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -490,13 +490,13 @@ def build_template_grid(templates,sort_order="DESC",sort_by="name")
   def dialog_partial_for_workflow
     case (@edit || @options).try(:[], :wf)
```

Comment:
This is a tiny method, please write a test for this so we can figure what this is doing... I don't understand why this case statement does this.